internal/pkg/deploy/cloudformation/stack/lb_web_service_integration_test.go

```diff
@@ -114,6 +114,7 @@ func TestLoadBalancedWebService_TemplateInteg(t *testing.T) {
 			},
 		})
 		serializer.SCFeatureFlag = true
+		serializer.EnvAddonsFeatureFlag = true
```

Comment:
I've changed the feautre flag to be container-level!

Reply:
Interesting 🤔 do you think there is a benefit for container level? I feel like the previous setup was simpler and equivalent.
Sidecars should still be able to reference `EnvAddonsFeatureFlag` and decide whether a field should be supported or not

Reply:
> Sidecars should still be able to reference EnvAddonsFeatureFlag and decide whether a field should be supported or not

This is not quite the case unless we change how we organize the `partials`s, or modify `include` template function such that it is able to handle multiple parameters. I am not sure if the benefit overcomes the amount of application code that needs to be made.

For example, when the feature flag is service-level, the sidecar section won't be able to read the feature flag data because what's passed into the `envvars-container`  partial template is the sidecar container's data, not the service's.